.travis.yml

```diff
@@ -7,6 +10,10 @@ python:
   - '3.6'
   - '3.5'
 before_install:
+  - |
+    if [[ "${TRAVIS_CPU_ARCH}" == "ppc64le" ]]; then
+      sudo chown -Rvf $USER:$GROUP ~/.cache/pip/wheels;
```

Comment:
Could you add, somewhere in the commit message, exactly what problem this is trying to solve? As in, why is this directory not owned by the user, and why does that happen only on ppc64le?

Also, the mixture of `-v` and `-f` is weird enough to merit a comment (or removal). Maybe an explanation in the commit message, if it's hard to add comments in the yaml. Thanks!

Reply:
I agree with @edsantiago, even an inline comment would be welcome.

Reply:
I also agree FWIW.

Reply:
Done